lib/timers.js

```diff
@@ -45,6 +44,10 @@ function insert(item, msecs) {
     list.msecs = msecs;
     list[kOnTimeout] = listOnTimeout;
   }
+  // Call start regardless whether list is new
+  // or not to prevent incorrect active_handles
+  // count. See https://github.com/nodejs/node-v0.x-archive/issues/25831.
+  list.start(msecs, 0);
```

Comment:
What happens if the previous list start time was less than this list start time? I think it should check that, if I am following the logic correctly.